### PR TITLE
Fixed formatting issues

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

This is how these issues are displayed on symfony.com right now:

issue1

---

issue2

---

issue3

---

issue4

---

issue5

### CODE REVIEW

File: cookbook/controller/service.rst [hunk 1]
```diff
@@ -277,8 +277,7 @@ controller:
         in the :class:`Symfony\\Component\\Routing\\Generator\\UrlGeneratorInterface`.
 
 :method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::getDoctrine` (service: ``doctrine``)
-
-    *Simply inject doctrine instead of fetching it from the container*
+    Simply inject doctrine instead of fetching it from the container.
```
Comment (xabbuh) [inline cookbook/controller/service.rst]: Can we really not have italic text in a definition list?

Comment (javiereguiluz) [inline cookbook/controller/service.rst]: Yes, we can maintain the italic text. I removed it because it looked wrong to me. I've just restored it.

Comment (xabbuh) [inline cookbook/controller/service.rst]: So the issue actually was the blank line?

Comment (javiereguiluz) [inline cookbook/controller/service.rst]: Yes. That generated an ugly `blockquote` instead of the usual paragraph text.

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thanks again Javier!

**Pull request closed without merging**